.github/CODEOWNERS

```
@@ -3,4 +3,4 @@
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
 # default owners
-* @celo-org/core-services @obasilakis @eruizgar91
\ No newline at end of file
+* @celo-org/data-services @obasilakis @eruizgar91
```

```suggestion
* @celo-org/data-services
```
as they are already team members.